values_test.go

```diff
@@ -42,24 +42,6 @@ func TestGetValues(t *testing.T) {
 	require.Equal(t, fl["main.GitCommit"], fl["main.GitSummary"])
 }
 
-func TestGetValues_versionFlag(t *testing.T) {
```

Comment:
this test is valuable please don't delete it.

it would be good to have an unit test for `versionFromFile(dir)` here:

- when there's no VERSION file it will be empty
- when VERSION the function returns its value

Reply:
How odd, not sure why I deleted it.

Reply:
This bit has been addressed, IIUC.